packages/proto-signing/src/codec/cosmos/bank/v1beta1/bank.ts

```diff
@@ -0,0 +1,650 @@
+/* eslint-disable */
+import { Coin } from "../../../cosmos/base/v1beta1/coin";
```

Comment:
> It doesn't generate any index files, so we'd either have to do that ourselves or stick to the nested imports pattern.

Yeah, I also wanted that, I added a bunch of `index.ts` manually, until I stumbled upon https://github.com/stephenh/ts-proto/issues/167#issuecomment-737936654. I proposed a solution in the thread. I believe generating `index.ts` is a desirable feature.